app/helpers/list-item.js

```diff
@@ -0,0 +1,7 @@
+import { helper } from '@ember/component/helper';
```

Comment:
Why do you need a new helper for this ? Ember math helpers are already part of the project. Use that to increment the index by 1 and display. Remove this one .